test/e2e/common/cleanup.sh

```diff
@@ -16,12 +16,53 @@
 set -x # echo so that users can understand what is happening
 set -e # exit on error
 
+
+env="kind"
+
+if [ "$1" == "--env" ]; then
```

Comment:
What should happen if there is no `--env` on the command line? I think that a silent "nothing" is not the best answer.

Reply:
if the flag --env is missing, it would default to use its default parameter "kind". The e2e test for kind is already using the same script without providing any flags